metricbeat/tests/system/test_traefik.py

```diff
@@ -0,0 +1,36 @@
+import os
+import metricbeat
+import unittest
+import time
+
+
+class Test(metricbeat.BaseTest):
+
+    COMPOSE_SERVICES = ['traefik']
+
+    @unittest.skipUnless(metricbeat.INTEGRATION_TESTS, "integration test")
+    def test_health(self):
```

Comment:
there is a new method `check_metricset` in metricbeat.py. Using it could simplify your code here.